src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -468,14 +468,14 @@ class BinaryCompactObject : public DomainCreator<3> {
 
   struct RotationMap {
     static constexpr Options::String help = {
-        "Options for a time-dependent rotation map about the z axis"};
+        "Options for a time-dependent rotation map about an arbitrary axis."};
     using group = TimeDependentMaps;
   };
   /// \brief The initial value of the rotation angle.
   struct InitialRotationAngle {
-    using type = std::array<double, 3>;
+    using type = double;
     static constexpr Options::String help = {
-        "Rotation angle about each axis at initial time."};
+        "Rotation angle about the initial angular velocity vector."};
```

Comment:
Why can the initial rotation only be about the angular velocity?

Reply:
When using the axis-angle representation, the angle is about the axis. The axis here *is* the angular velocity, so the angle is about the angular velocity.

Reply:
But why do the value and derivative have to share the same axis?

Reply:
Now I'm questioning if we even need a rotation angle. There are a couple notions of "angle" here and I want to make sure we are talking about the same ones. A unit quaternion represented as q = (cos(theta/2), nhat*sin(theta/2)) will rotate a vector about nhat by an angle theta. Here, nhat is the angular velocity. So "theta" is one notion of angle about the angular velocity. Then there are phi_x, phi_y, phi_z where the x-component of the angular velocity omega_x = dt(phi_x), omega_y = dt(phi_y), etc. This is another notion. I was referring to "theta" in that description, but you seem to be referring to "phi_x" etc. when you say value and derivative.

For initial conditions, we must specify the initial quaternion and initial angular velocity to properly construct things. But since the vector part of the quaternion is just the unit vector in the direction of the angular velocity, we technically only need "theta" to specify the quaternion. After thinking about it, I think we can just set "theta", "phi_x", etc. = 0 always for initial conditions, so the initial quaternion is just (1,0,0,0). (For reference, for SpEC ID, you only specify omega_z and the quaternion is assumed to be (1,0,0,0), implying that theta = 0)

Also, for our initial conditions, are we ever going to start a binary where the angular velocity isn't initially aligned with the z-axis? If we are, then I can leave the InitialAngularVelocity as a vector. But if not, I think we can make it a scalar again and have it just be the z-component of the initial vector.

Thoughts?

Reply:
I don't think there's a reason to have more freedom here than in SpEC.  Maybe there are cases where a slightly off-axis initial rotation would be better, but I haven't heard about anyone having problems with the SpEC version so any such effect is probably small enough that the control system can quickly compensate.